content/applications/sales/sales/products_prices/prices/pricing.rst

```diff
@@ -353,6 +353,53 @@ discount for each product line and the new price will automatically be calculate
    Positive values for :guilabel:`Disc.%` will apply a *discount*, while negative values can be
    used to *increase* the price.
 
+Discount button
+---------------
+
+In Odoo 17, with the *Discounts* setting enabled, a :guilabel:`Discount` button appears at the
+bottom of sales orders.
+
+.. image:: pricing/discount-button-sales-order.png
+   :align: center
+   :alt: The discount button located at the bottom of a sales order form in the Odoo Sales app.
+
+When the :guilabel:`Discount` button on a sales order is clicked, a :guilabel:`Discount` pop-up
+window appears.
+
+.. image:: pricing/discount-popup.png
+   :align: center
+   :alt: The Discount pop-up window that appears when the Discount button is clicked.
+
```

Comment:
IMO this image is unnecessary, please consider removing to cut down on the number of images in this document :)
```suggestion
```